text_generation/causal_lm/cpp/continuous_batching/library/CMakeLists.txt

```diff
@@ -1,6 +1,7 @@
 # Copyright (C) 2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
+cmake_minimum_required(VERSION 3.15)
 # start of dependencies
 
```

Comment:
Is `nlohmann_json` needed here? The library does not use it as far as I know and for apps targets it's already defined in another cmake file.

Reply:
library uses it to read `generation_config.json`

it's header only library which is not visible once continuous batching is built